src/core/lib/attribute-transition-manager.js

```diff
@@ -0,0 +1,367 @@
+import {GL, Program, Model, Geometry, Buffer, TransformFeedback} from 'luma.gl';
+
+const ATTRIBUTE_MAPPING = {
+  1: 'float',
+  2: 'vec2',
+  3: 'vec3',
+  4: 'vec4'
+};
+
+const TRANSITION_STATE = {
+  NONE: 0,
+  PENDING: 1,
+  STARTED: 2,
+  ENDED: 3
+};
+
+const noop = () => {};
+
+class ProgramTransformFeedback extends Program {
```

Comment:
This should to be sorted with luma.gl before we land. Looks like a copy of a hack from the Wind example which predates the current [luma.gl version](https://github.com/uber/luma.gl/blob/master/src/webgl/program.js#L64)